7.4.цикл_while/7.4.8.py

```diff
@@ -0,0 +1,4 @@
+n = input()
+while (n!='КОНЕЦ'):  
```

Comment:
не забываем про PEP8, так же тут не нужны скобки:
```suggestion
while n != 'КОНЕЦ':
```